sync/_databricks.py

```diff
@@ -1363,55 +1391,103 @@ def _event_log_poll_duration_seconds():
     return 15
 
 
-def _get_eventlog_from_s3(
+def _s3_eventlog_prefix(base_filepath: str, cluster_id: str) -> str:
+    # If the event log destination is just a *bucket* without any sub-path, then we don't want to include
+    #  a leading `/` in our Prefix (which will make it so that we never actually find the event log), so
+    #  we make sure to re-strip our final Prefix
+    # TODO - using the spark_context_id might be good here, as we do in the DBFS logic
+    return f"{base_filepath}/eventlog/{cluster_id}".strip("/")
```

Comment:
```suggestion
    return f"{base_filepath}/eventlog/{cluster_id}".lstrip("/")
```

if we're committed to striping only at beginning